packages/admin/cms-admin/src/generator/future/generateForm/generateFields.ts

```diff
@@ -12,6 +12,7 @@ export type GenerateFieldsReturn = GeneratorReturn & {
     formValueToGqlInputCode: string;
     formValuesConfig: {
         omitFromFragmentType?: string;
+        destructFromFormValues?: string; // equals omitting from formValues copied directly to mutation-input
```

Comment:
Alternatively you could name this omitFromGqlInput

Reply:
`omitFromGqlInput` was the name before. While implementing some other feature I used `omitFromGqlInput` and needed to debug/check why the input was still modified (there is `formValueToGqlInputCode` to modify the gqlInput). So I decided the name is misleading and renamed it.